store/sql_store.go

```diff
@@ -127,6 +123,11 @@ func setupConnection(con_type string, driver string, dataSource string, maxIdle
 }
 
 func (ss SqlStore) DoesColumnExist(tableName string, columnName string) bool {
+	// XXX TODO FIXME this should be removed after 0.6.0
```

Comment:
I've also added a comment to MM-1698 to remove the alter table methods but leave the index creation method.